Module/PSWordCloud.psd1

```diff
@@ -0,0 +1,129 @@
+
+    # Copyright statement for this module
+    Copyright            = '(c) 2019 Joel Sallow (/u/ta11ow, @vexx32). All rights reserved.'
+
+    # Description of the functionality provided by this module
+    Description          = 'Turn your scripts and documents into pretty and practical word clouds!'
+
+    # Minimum version of the Windows PowerShell engine required by this module
+    PowerShellVersion    = '6.1'
+
+    # Name of the Windows PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the Windows PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # CLRVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = @()
+
+    # Modules that must be imported into the global environment prior to importing this module
+    # RequiredModules       = @()
+
+    # Assemblies that must be loaded prior to importing this module
+    # RequiredAssemblies = @()
+
+    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
+    # ScriptsToProcess      = @()
+
+    # Type files (.ps1xml) to be loaded when importing this module
+    # TypesToProcess = @()
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    # NestedModules = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport    = @()
+
+    # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+    CmdletsToExport      = @('New-WordCloud')
+
+    # Variables to export from this module
+    VariablesToExport    = '*'
+
+    # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+    AliasesToExport      = @('wordcloud', 'wcloud', "nwc")
+
+    # DSC resources to export from this module
+    # DscResourcesToExport = @()
+
+    # List of all modules packaged with this module
+    # ModuleList = @()
+
+    # List of all files packaged with this module
+    FileList             = @(
+        'Private/Get-ImageFormat.ps1'
+        'Private/ConvertTo-Radians.ps1'
+        'Public/New-WordCloud.ps1'
+        'PSWordCloud.psd1'
+        'PSWordCloud.psm1'
+        'Data\StopWords.txt'
```

Comment:
You deleted `data\stopwords.txt` as part of this PR, and replaced it with a big array and so I would recommend you consider removing it from this list. From the look of it, I'm not sure this list is still accurate in other ways, as well 😁